26-register_registration/creator_navigates_to_resource_type_tab.feature

```diff
@@ -0,0 +1,408 @@
+Feature: Creator navigates to Resource Type tab
+
+  @274
+  Scenario: Creator navigates to the Resource Type tab and selects Reference type "Contribution to journal"
```

Comment:
```suggestion
  Scenario: Creator navigates to the Resource Type tab and selects Resource type "Contribution to journal"
```

Reply:
jeg antok at det fortsatt skal være Reference her (som i verdien), og at bare taben skulle renames... dette er en verdi fra datamodellen og der ligger den i `reference`, men ble usikker nå 🤔

Reply:
Skjønner hva du tenker, men synes ikke det gir mening å kalle det `Reference type` lengre... For den typen var jo knyttet til taben som het `Reference`...

Reply:
jeg tenker heller at den er knyttet til verdien vi lagrer, som er i `entityDescription.reference`, men det burde sikkert eeegentlig være likt navn i datamodell og tab-fane da. Uansett vi som skal skjønne scenarioene, så det gir kanskje mest mening å bare rename dette også til `Resource Type` ja